src/lib.rs

```diff
@@ -11,3 +11,7 @@ pub mod mysql;
 pub mod postgres;
 #[cfg(feature = "sqlite")]
 pub mod sqlite;
+
+pub(crate) fn path_has_absolute_sequence(path: &str) -> bool {
```

Comment:
This would be more robust if it used the Rust `Path` type. It could get the current directory, call https://doc.rust-lang.org/std/path/struct.Path.html#method.canonicalize on the input path and verify the final path is within the current directory. And this function could return `Result<>` directly, preventing the calling functions from needing to wrap it.